apps/aecore/src/aec_base58c.erl

```diff
@@ -63,8 +63,8 @@ split(Bin) ->
     binary:split(Bin, [<<"$">>], []).
 
 check_str(Bin) ->
-    <<C:32/bitstring,_/binary>> = crypto:hash(sha256,
-                                              crypto:hash(sha256, Bin)),
+    <<C:32/bitstring,_/binary>> =
+        aec_hash:blake2s_hash(aec_hash:blake2s_hash(Bin)),
```

Comment:
Is this correct though? The base58Check encoding is defined as using SHA256.
https://en.bitcoin.it/wiki/Base58Check_encoding